packages/manager/src/managers/prismicRepository/PrismicRepositoryManager.ts

```diff
@@ -423,6 +424,29 @@ export class PrismicRepositoryManager extends BaseManager {
 		}
 	}
 
+	async fetchEnvironments(): Promise<Environment[]> {
+		const repositoryName = await this.project.getRepositoryName();
+
+		const url = new URL("./environments", API_ENDPOINTS.PrismicWroom);
```

Comment:
`PrismicCoreAPI` rather than `PrismicWroom`

Reply:
We don't have a `PrismicCoreAPI` value currently. What URLs should be used for production, staging, and development?

For reference, here where we define the URLs: https://github.com/prismicio/slice-machine/blob/7c995fe301c4c7ae41faa433a339ce6cb3340ceb/packages/manager/src/constants/API_ENDPOINTS.ts#L14-L77

Reply:
I'm still working on the lambda so I don't have production URL yet :/
I just thought PrismicCoreAPI is a more explicit name